metabase/metabase
# Preserve filters during navigating back to dashboard

Related to https://github.com/metabase/metabase/issues/31786

### Description

Preserve filters during navigating back to dashboard

### How to verify

There is an e2e test, but manually

1. open dashboard with a question
2. set filter value to any
3. navigate to a question
4. click back button to return back to dashboard
5. filter value should have the previous value

### Demo

https://www.loom.com/share/8b1ebde1f23e428b915e60e9b3c9c59e

### Checklist
